op-service/cliapp/lifecycle_test.go

```diff
@@ -85,11 +87,14 @@ func TestLifecycleCmd(t *testing.T) {
 		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 
 		// puppeteer system signal interrupts by hooking up the test signal channel as "blocker" for the app to use.
-		ctx = opio.WithBlocker(ctx, func(ctx context.Context) {
+		ctx = ctxinterrupt.WithWaiterFunc(ctx, func(ctx context.Context) (interrupt, ctxErr error) {
```

Comment:
Refactored this so you could signal an interrupt cause without competing with the Context cancellation